ts/utils/input.ts

```diff
@@ -20,6 +20,9 @@ export const CreditCardPan = PatternString(
 );
 export type CreditCardPan = t.TypeOf<typeof CreditCardPan>;
 
+export const CreditCardHolder = PatternString(`^([\x20-\x5f\x61-\x7d]*)$`);
```

Comment:
```suggestion
export const CreditCardHolder = PatternString(`^([\x20-\x5f\x61-\x7d]+)$`);
```
@CrisTofani is an empty string a valid card holder? Because that regex makes it pass.

Reply:
[6fbcd39](https://github.com/pagopa/io-app/pull/4034/commits/6fbcd390790dfc0fb6bf8b9605df409dc3816c9d) should fix! good catch